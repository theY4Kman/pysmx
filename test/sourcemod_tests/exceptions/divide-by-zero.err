Error executing main: Divide by zero
