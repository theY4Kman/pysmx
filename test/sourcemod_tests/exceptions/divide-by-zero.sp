// returnCode: 1
public main()
{
  int a = 5;
  int b = 0;
  return a / b;
}
