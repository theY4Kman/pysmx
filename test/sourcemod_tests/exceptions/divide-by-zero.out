Exception thrown: Divide by zero
  [0] divide-by-zero.sp::main, line 6
